client/client-multi/client-c/CMakeLists.txt

```diff
@@ -292,10 +275,20 @@ add_subdirectory(Modules/doxygen)
 
 add_subdirectory(Modules/cppcheck)
 
-externalproject_add(kaa_rsa_keygen SOURCE_DIR "${CMAKE_CURRENT_LIST_DIR}/tools/kaa_encryption"
-                    INSTALL_COMMAND  "./generate_rsa_keys" COMMENT "Generating RSA headers"
-                    BINARY_DIR ${CMAKE_CURRENT_BINARY_DIR}/kaa_keys)
-
-externalproject_add_step(kaa_rsa_keygen "Copying generated files"
-                         COMMAND mv -f "${CMAKE_CURRENT_BINARY_DIR}/kaa_keys/kaa_keys_gen.h"
-                         "${KAA_SRC_FOLDER}/gen" DEPENDEES install)
+if(${KAA_PLATFORM} STREQUAL "posix")
```

Comment:
Let's also leave heavy comment that documents what happens here. Without a proper explanation it looks hackish.